changelog.d/5897.feature

```diff
@@ -0,0 +1 @@
+Switch to the v2 lookup API for 3PID invites.
```

Comment:
I suggest mentioning the MSC which this is implementing. (Also worth linking to it from the PR description).

Reply:
(seems to be 2134, plus the auth bits from 2140?)

Reply:
Good point, and yes that's correct.